### PR TITLE
Repository URL support.

### PR DESCRIPTION
**Pull request by edgonmsft**
Documentation for upcoming support for Pull Request urls.

### CODE REVIEW

File: vs-online/reference/repository.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ms.author: edgonmsft
+ms.service: visual-studio-online
+title: Repository Options Visual Studio Online
+ms.topic: overview
+ms.date: 02/14/2020
+---
+
+# Repository support
+
+Visual Studio Online's [environments](../overview/what-is-vsonline.md#environments) can be initialized with a git repository. A [devcontainer.json](configuring.md) file contained inside the repository also provides additional configuration opportunities.
```
Comment (nikmd23) [inline vs-online/reference/repository.md]:
```suggestion
Visual Studio Online's [environments](../overview/what-is-vsonline.md#environments) can be initialized with a Git repository. A [devcontainer.json](configuring.md) file contained inside the repository also provides additional configuration opportunities.
```

File: vs-online/reference/repository.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Repository support is handled using the git clone url obtained from your source control provider.
```
Comment (nikmd23) [inline vs-online/reference/repository.md]:
```suggestion
Repository support is handled using the Git clone url obtained from your source control provider.
```

File: vs-online/reference/repository.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Any URL that can be cloned directly locally on your machine should work in this way.
+
+## Private Repositories
+
+### VSCode
```
Comment (nikmd23) [inline vs-online/reference/repository.md]:
```suggestion
### Visual Studio Code extension
```

File: vs-online/reference/repository.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Our Visual Studio Code extension uses your local git installation to obtain credentials to perform clone operations. We do not save your credentials anywhere.
```
Comment (nikmd23) [inline vs-online/reference/repository.md]:
```suggestion
Our Visual Studio Code extension uses your local Git installation to obtain credentials to perform clone operations. We do not save your credentials anywhere.
```

File: vs-online/reference/repository.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Support is straight forward, if you can perform operations locally they should just work.
+
+### Browser
+
+Our browser support performs OAuth flows for the selected provider of the git url provided. We are continuously working to improve and extend our support for additional providers.
```
Comment (nikmd23) [inline vs-online/reference/repository.md]:
```suggestion
Our browser support performs OAuth flows for the selected repository hosting provider of the Git url given. We are continuously working to improve and extend our support for additional hosting providers.
```

File: vs-online/reference/repository.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+For now we support the following:
```
Comment (nikmd23) [inline vs-online/reference/repository.md]:
```suggestion
We currently support the following:
```

**Comment by nikmd23** [comment]
The docs team (@conceptdev, @davidbritch & @profexorgeek) have been given access to this repo. They should also do a review before we merge this PR.

**Comment by davidbritch** [comment]
@nikmd23 

I've edited this PR.

I'd also recommend:

1. Updating the page title, for the reasons I outlined on the other PR.
1. Adding a link on the first use of "Visual Studio Code extension" to the extension download.

**Pull request merged**